
{% extends "layout.html" %}
  
{% block title %}Design{% endblock %}
{% block lead %}Expand upon your Silver medal work for Proposed Implementation and develop a proof of concept for your project.{% endblock %}

{% block page_content %}

<!-- <div class="row mt-4">
  <div class="col">
    <div class="bd-callout bd-callout-info">
      <h4>Gold Medal Criterion #4</h4>
      <p>Expand upon your Silver medal work for Proposed Implementation and develop a proof of concept for your project.<p>
      <hr>
      <p>Please see the <a href="https://competition.igem.org/judging/medals">2022 Medals Page</a> for more information.</p>
    </div>
  </div>
</div> -->

<style>
.mhyrightcontent img {
  width:80%;
  height:80%;
}
</style>
<div>
  <!-- Pagination_header -->
  <header>
  <img src="https://static.igem.wiki/teams/4308/wiki/static/imgs/design.png"  class="mhypntopimg" alt="图片加载失败" >
  <img src="https://static.igem.wiki/teams/4308/wiki/static/imgs/cat.png"  class="mhycatimg" alt="图片加载失败" >
  </header>
  <div class="mhypnmain">
      <!-- left -->
      <div class="mhypnleft">
          <div class="mhyleftnavtitle">Design</div>
          <div class="mhyleftnav">
              <ul class="mhyleftcatlog">
                  <li><a href="#title1"> Overview </a></li>
                  <li><a href="#title2"> Detection Platform </a></li>
                  <li><a href="#title3">  </a></li>
                  <li><a href="#title4">  </a></li>
                  <li><a href="#title5">  </a></li>
                  <li><a href="#title6">  </a></li>
              </ul>
          </div>
          <img src="https://static.igem.wiki/teams/4308/wiki/static/imgs/dna.png" class="mhyDNAimg" alt="">
      </div>
      <!-- right -->
      <div class="mhypnright">
          <div class="mhyrightcontent">

            <!-- 内容 -->
            <h2 id='title1'>Overview</h2>
            <p>To provide health security for individuals in disadvantaged areas, our team works on early cancer screening and preliminary identification of different malignancies. We have constructed a time-saving, easy-to-use and low-cost platform for detection of cancer related biomarkers, which can be applied to primary medical institutions and home testing. We employed bioinformatics to identify a lncRNA molecule that is strongly correlated to the occurrence and progression of more than 20 different types of cancer. The detection platform is made up of four parts in total, including RT-RPA, CRISPR-Cas14a, DNA nanosponge and electrochemical analysis. Target RNA molecules can be efficiently amplified by RT-RPA reaction, and then trigger the trans-cleavage activity of Cas14a specifically. The trans-cleavage of Cas14a will collapse the DNA nanosponge to release the γ-amylase trapped in the nanosponge. Finally, γ-amylase catalyzes the hydrolysis of amylose to produce glucose, which could be detected with a personal glucose meter. As a result, this platform can quantitatively analysis the expression of lncRNA in human blood without the need of costly devices and the whole test process requires only a few manual operations, which can be completed within 1 hour. It is a valuable tool for assessing the risk of cancer.</p>

            <figure style="display: flex; justify-content: center; align-items: center;">
                 <img src="https://static.igem.wiki/teams/4308/wiki/static/imgs/design/picture1.png" width="60%" style="float:center;"> 
              </figure> 
            
            <span><strong>Figure .1</strong> Our platform design for detecting biomarker </span>
            <h2 id='title2'>Detection Platform</h2>
            <h4 id='biomarker'>Biomarker</h4>
            <p>In the search for detection biomarker, our model found differential expression of LINC00857 in 25 cancers by comparing the expression of this molecule in each tumor group and normal group using TCGA combined with GTEx database. Then, we further explored the application of LINC00857 in cancer staging. Using the GEPIA2 database, we found that it was also differentially expressed in the staging of multiple cancers, such as endometrial cancer, colon cancer, esophageal cancer, pancreatic cancer, and cutaneous melanoma. As shown in (<strong>Figure 2</strong>), the expression level of LINC00857 in some tumor tissues higher than the corresponding control tissues, indicating the potential of LINC 00857 as a cancer staging marker. At the same time, the molecule correlates with the prognosis of cancer patients, which means that by detecting the expression of this lincRNA, we can not only achieve the primary screening of many cancers, but also evaluate prognosis of some cancer patients. Therefore, we chose LINC00857 as our detection biomarker, and designed the supporting detection system.</p>

            <figure style="display: flex; justify-content: center; align-items: center;">
                 <img src="https://static.igem.wiki/teams/4308/wiki/static/imgs/design/picture2.png" width="60%" style="float:center;"> 
              </figure> 
           
            <span><strong>Figure .2</strong> Expression level of LINC000857 in TCGA tumors vs. adjacent tissues (if available) as visualized by GEPIA2.</span>
            <h4 id='isothermal-amplification'>Isothermal amplification</h4>
            <p>Due to its excellent efficiency and convenience, we selected Reverse Transcription-Recombinase Polymerase Amplification (RT-RPA) to amplify the target RNA <sup>[1]</sup> . The target sequence in LINC00857 is first converted into cDNA under the action of reverse transcriptase. The recombinase-primer complex interrogates the cDNA seeking a homologous sequence and promotes strand exchange by the primer at the cognate site. And the displaced DNA strand is stabilized by single-stranded binding proteins. Then, the recombinase disassembles and a strand displacing DNA polymerase binds to the 3’ end of the primer to elongate it in the presence of dNTPs. Cyclic repetition of this process results in the achievement of exponential amplification<sup>[2]</sup>. In this system, we used in-vitro transcription to prepare artificial target sequence of LINC00857 RNA. Since we did not utilize human materials (blood) in the experiment. We design a pair of primers that can accomplish rapid and effective amplification within 10 minutes at 42 ℃, which further indicates its fast, convenient and efficient performance. Here follows a video of the RPA principle.</p>

            <figure style="display: flex; justify-content: center; align-items: center;">
                 <img src="https://static.igem.wiki/teams/4308/wiki/static/imgs/design/figure1.gif" width="60%" style="float:center;"> 
              </figure> 

              <figure style="display: flex; justify-content: center; align-items: center;">
                 <img src="https://static.igem.wiki/teams/4308/wiki/static/imgs/design/picture3.png" width="60%" style="float:center;"> 
              </figure> 

            
            <span><strong>Figure 3.</strong> Principle of Recombinase Polymerase Isothermal Amplification (RT-RPA)</span>
            <h4 id='crispr-cas14a'>CRISPR-Cas14a</h4>
            <p>CRISPR-Cas systems are potential biosensing technologies for nucleic acid detection as a result of the recent discovery of a family of Cas nucleases with trans-cleavage activity.We chose Cas12a, Cas14a, and CasΦ from the Cas12 family as three candidates for our system. Among them, the most researched one is Cas12a, which has been extensively used in in vitro detection systems because of its effectiveness. The CRISPR-Cas14a system is currently the smallest functioning CRISPR system discovered, which has a better single-base mismatch recognition ability compared with Cas12a, makes it suitable for precise nucleic acid detection<sup>[3]</sup> . When performing nucleic acid splicing, it is independent of the PAM sequence and is capable of detecting single base mismatches,which makes it suitable for precise detection. CasΦ, a newly reported Cas nuclease, also has a good target recognition specificity, and its size is only half of Cas12a<sup>[4]</sup> . We finally settle on Cas14a as the connection between RT-RPA and DNA nanosponge in the system by contrasting Cas14a with CasΦ and Cas12a activity verification.</p>
            <p>By turning on its trans-cleavage activity after attaching to single or double stranded target amplification products, Cas14a can nonspecific cutting of ssDNA molecules. Therefore, we made the decision to use it to decompose the DNA nanosponge. When our amplified product activated Cas14a (<strong>Figure 4</strong>), it cleaved single-stranded DNA randomly, causing the DNA nanostructure to disintegrate and the release of γ-amylase, which catalyzed the hydrolysis of amylose and generated a significant amount of glucose. Overall, the Cas14a plays the role of signal hub, integrating upstream information for our next parts.</p>

              <figure style="display: flex; justify-content: center; align-items: center;">
                 <img src="https://static.igem.wiki/teams/4308/wiki/static/imgs/design/picture4.png" width="60%" style="float:center;"> 
              </figure> 
            
            <span><strong>Figure 4.</strong> Cas14a is activated and cleaves the F-Q reporter.</span>
            <h4 id='dna-nanosponge'>DNA Nanosponge</h4>
            <p>We refer to the hydrogel-based enzyme encapsulation strategy of Chaoyong James Yang <sup>[5]</sup> and YiYang <sup>[6]</sup>  of Chaoyong. In order to make this part into lyophilized preparation for easy preservation, we have designed γ- Amylase coated DNA nanosponge, which is constructed by RCA (rolling loop amplification) amplification system and MBD-SA fusion protein structure (<strong>Figure 5</strong>).</p>
            <p>MBD is a family of methyl-CpG-binding domain proteins, known to include 11 proteins. And all of them contain a highly conserved protein domain, named MBD domain [7]. In our design, the MBD-SA fusion protein with 4 MBD structural domains recognizes and binds the repetitive CpG sites (CGAT CCGCGGC TCTCTC GCCGCGG ATCG) in the stem-loop structure in RCA-amplified DNA. Then, MBD-SA binds ssDNA and forms DNA nanostructure by self-assembling wrapped amylase. When Cas14a was activated, it can cleave DNA nanosponge, thereby releasing γ-amylase to break down free straight-chain starch for glucose detection(<strong>Figure 6</strong>).</p>

            <figure style="display: flex; justify-content: center; align-items: center;">
                 <img src="https://static.igem.wiki/teams/4308/wiki/static/imgs/design/picture5.png" width="60%" style="float:center;"> 
              </figure> 
            
            <span><strong>Figure 5.</strong> Process of RCA and DNA nanostructure formation</span>

            <figure style="display: flex; justify-content: center; align-items: center;">
                 <img src="https://static.igem.wiki/teams/4308/wiki/static/imgs/design/picture6.png" width="60%" style="float:center;"> 
              </figure> 

            
            <span><strong>Figure 6.</strong>Process of Cas14a collapses the DNA nanosponge</span>
            <h4 id='electrochemical-detection'>Electrochemical detection</h4>
            <p>We selected glucose created in the previous phase as the biological signal since its concentration is closely connected to the quantity of our targets. We designed a novel hardware framework that intuitively translate biological inputs into detection outcomes.</p>
            <p>A redox reaction will take place and current will be produced when the glucose made by the nanosponge is exposed to the glucose oxidase contained in the electrode. The hardware receives amplified electric signal which redox reaction produces in order to carry out relevant analysis subsequently. This signal will be evaluated in the central chip and converted to output by a preset program. The findings can be seen right away on the LED and LCD display, and the Bluetooth module can transfer the data to a smartphone. Here follows a video of electrochemical detection of glucose concentration.</p>
            

            <h4 id='reference'>Reference</h4>
            <p class="mhyref">[1] Piepenburg O, Williams C H,Stemple D L, et al. DNA Detection Using Recombination Proteins[J]. PLoS Biology, 2006, 4(7):e204.</p>
            <p class="mhyref">[2] Jiao Y, Xu C, Li J, Gu Y, Xia C, Xie Q, Xie Y, An M, Xia Z, Wu Y. Characterization and a RT-RPA assay for rapid detection of Chilli Veinal mottle virus (ChiVMV) in tobacco. Virol J. 2020 Mar 10;17(1):33. doi: 10.1186/s12985-020-01299-w. PMID: 32156292; PMCID: PMC7065361.</p>
            <p class="mhyref">[3] Harrington LB, Burstein D, Chen JS, Paez-Espino D, Ma E, Witte IP, Cofsky JC, Kyrpides NC, Banfield JF, Doudna JA. Programmed DNA destruction by miniature CRISPR-Cas14 enzymes. Science. 2018 Nov 16;362(6416):839-842. doi: 10.1126/science.aav4294. Epub 2018 Oct 18. PMID: 30337455; PMCID: PMC6659742.</p>
            <p class="mhyref">[4] Pausch, P., B. Al-Shayeb, E. Bisom-Rapp, et al., CRISPR-CasPhi from huge phages is a hypercompact genome editor. Science, 2020. 369(6501): p. 333-337.doi: 10.1126/science.abb1400. </p>
            <p class="mhyref">[5]Yan L, Zhu Z, Zou Y, Huang Y, Liu D, Jia S, Xu D, Wu M, Zhou Y, Zhou S, Yang CJ. Target-responsive &quot;sweet&quot; hydrogel with glucometer readout for portable and quantitative detection of non-glucose targets. J Am Chem Soc. 2013 Mar 13;135(10):3748-51. doi: 10.1021/ja3114714. Epub 2013 Jan 29. PMID: 23339662.</p>
            <p class="mhyref">[6]Song J , He W , Shen H , Zhou Z , Li M , Su P , Yang Y . Self-assembly of a magnetic DNA hydrogel as a new biomaterial for enzyme encapsulation with enhanced activity and stability. Chem Commun (Camb). 2019 Feb 21;55(17):2449-2452. doi: 10.1039/c8cc09717h. PMID: 30734040.</p>
            <p class="mhyref">[7] Jeltsch A, Broche J, Lungu C, et al. Biotechnological Applications of MBD Domain Proteins for DNA Methylation Analysis[J]. Journal of Molecular Biology, 2020, 432(6): 1816-1823.</p>
                   

          
          </div>
      </div>
  </div> 
</div>


{% endblock %}
